name_mark.rb

```diff
@@ -1,4 +1,4 @@
- hash = {}
+hash = {}
 for i in (1..10)
```

Comment:
Try to avoid for loop - use each, times or the similar iterators.

Reply:
I changed into for loop to times. Can you check it?